app/code/Magento/InventoryCatalog/Test/Integration/IsSingleStockModeTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryCatalog\Test\Integration;
+
+use Magento\InventoryApi\Api\SourceRepositoryInterface;
+use Magento\InventoryCatalog\Model\IsSingleStockModeInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+class IsSingleStockModeTest extends TestCase
+{
+    /**
+     * @var IsSingleStockModeInterface
+     */
+    protected $isSingleStockMode;
+
+    /**
+     * @var SourceRepositoryInterface
+     */
+    protected $sourceRepository;
+
+    /**
+     * @inheritdoc
+     */
+    protected function setUp()
+    {
+        $this->isSingleStockMode = Bootstrap::getObjectManager()->get(IsSingleStockModeInterface::class);
+        $this->sourceRepository = Bootstrap::getObjectManager()->get(SourceRepositoryInterface::class);
+    }
+
+    public function testIsSingleStockModeOnCleanInstall()
+    {
+        self::assertCount(1, $this->sourceRepository->getList()->getItems());
```

Comment:
`self::assertCount(1, $this->sourceRepository->getList()->getItems());`
This check is redundant
We don'nt need to check qty of all sources. Second check will be enough